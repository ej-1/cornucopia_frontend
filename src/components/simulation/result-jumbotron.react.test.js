import React from "react";
import ResultJumbotron from "../simulation/result-jumbotron";
import renderer from "react-test-renderer";

test("InfoBox gets rendered", () => {
  let roi = 0.2345;

  const component = renderer.create(<ResultJumbotron roi={roi} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
