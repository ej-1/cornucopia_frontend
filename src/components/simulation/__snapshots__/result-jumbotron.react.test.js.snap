// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox gets rendered 1`] = `
<div
  className="result-jumbotron-container"
>
  <div
    className="result-jumbotron"
  >
    <h3>
      Total ROI: 
      23.45
       %
    </h3>
  </div>
</div>
`;
